README.md

```diff
@@ -78,21 +78,33 @@ This project uses [CMake] and [Hunter] package manager.
    mkdir build; cd build
    ```
 
-2. Configure the project with CMake. Check out additional
+1. Configure the project with CMake. Check out additional
    [configuration options](#cmake-configuration-options).
 
    ```sh
    cmake ..
    ```
+   e.g. incantation for Ubuntu 17.04 "Zesty" with CUDA, without CL, and addressing the
+   problems of requiring an older version of gcc and error "relocation R_X86_64_32S
+   against.bss' can not be used when making a shared object" mentioned in [this
+   issue](https://github.com/ethereum-mining/ethminer/issues/38).
+   ```sh
+   sudo apt install nvidia-{cuda-toolkit,opencl-dev} g{cc,++}-4.9 && \
```

Comment:
I think GCC 5 is fine.
Also, can you move the CUDA installation into a separated, optional build step?

Reply:
Confirmed, I was able to build with gcc-5. I'd misread the error before.
```
In file included from /usr/include/cuda_runtime.h:78:0,
                 from <command-line>:0:
/usr/include/host_config.h:119:2: error: #error -- unsupported GNU version! gcc versions later than 5 are not supported!
 #error -- unsupported GNU version! gcc versions later than 5 are not supported!
  ^~~~~
CMake Error at ethash-cuda_generated_ethash_cuda_miner_kernel.cu.o.Release.cmake:222 (message):
```